stake-pool/js/test/mocks.ts

```diff
@@ -132,6 +132,69 @@ export function mockTokenAccount(amount = 0) {
   };
 }
 
+const mockRpc = (data: any) => {
+  const value = {
+    owner: 'Stake11111111111111111111111111111111',
+    lamports: LAMPORTS_PER_SOL,
+    data,
+    executable: false,
+    rentEpoch: 0,
+  };
+  const result = {
+    context: {
+      slot: 11,
+    },
+    value: value,
+  };
+  return result;
+};
+
+export function mockStakeAccount(): any {
+  const data = {
+    program: 'stake',
+    parsed: {
+      type: 'delegated',
+      info: {
+        meta: {
+          rentExemptReserve: new BN(1),
+          lockup: {
+            epoch: 32,
+            unixTimestamp: 2,
+            custodian: new PublicKey(12),
+          },
+          authorized: {
+            staker: new PublicKey(12),
+            withdrawer: new PublicKey(12),
+          },
+        },
+        stake: {
+          delegation: {
+            voter: new PublicKey(
+              new BN('e4e37d6f2e80c0bb0f3da8a06304e57be5cda6efa2825b86780aa320d9784cf8', 'hex'),
+            ),
```

Comment:
nit: for consistency, please hardcode this like the other one and / or consider something simpler!